tests/unit/services/resolvers/bonfida-resolver.service.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { BonfidaResolverService } from "@resolver/services/resolvers/bonfida-resolver.service";
+import SolWeb3Service from "@resolver/services/web3/sol-web3.service";
+import { Cluster, Connection, PublicKey } from "@solana/web3.js";
+import { getDomainKeySync, NameRegistryState } from "@bonfida/spl-name-service";
+
+jest.mock("@bonfida/spl-name-service");
+const mockedGetDomainKey = jest.mocked(getDomainKeySync);
+mockedGetDomainKey.mockReturnValue({
+    isSub: false,
+    pubkey: new PublicKey("4DbiZPib1MvFZAecn8rQZtfVHiVQLFGwFTk2ZUawyG2i"),
+    parent: new PublicKey("4DbiZPib1MvFZAecn8rQZtfVHiVQLFGwFTk2ZUawyG2i"),
+    hashed: new Buffer("123")
+});
+
+jest.spyOn(SolWeb3Service, "getWeb3").mockImplementation((cluster: Cluster) => {
+    return {} as Connection
+})
+
+describe('bonfida-resolver.service', () => {
+    const bonfidaResolverService = new BonfidaResolverService();
+
+    const spyRetrieve = jest.spyOn(NameRegistryState, 'retrieve');
+
+    beforeEach(() => {
+        spyRetrieve.mockReset();
+    })
+
```

Comment:
Add following tests:
- SHOULD does not get addresses for domain with network IF is invalid
- SHOULD correct handle errors